Whiskey/Tasks/NodeLicenseChecker.ps1

```diff
@@ -22,8 +22,14 @@ function Invoke-WhiskeyNodeLicenseChecker
 
     Write-WhiskeyDebug -Context $TaskContext -Message ('Generating license report')
     $reportJson = Invoke-Command -NoNewScope -ScriptBlock {
-        & $nodePath $licenseCheckerPath '--json'
+        & $nodePath $licenseCheckerPath '--json' '--failOn' 'AGPL-1.0-or-later;GPL-1.0-or-later;LGPL-2.0-or-later'

```

Comment:
Let's make this an optional `FailOn` task property where people can define their own licenses to fail on.